locale/en/LC_MESSAGES/django.po

```diff
@@ -3867,6 +3871,14 @@ msgstr "Assess next submitter manually"
 msgid "ADD_ONE_EXTRA_SUBMISSION_FOR_STUDENT_IN_EXERCISE"
 msgstr "Add one extra submission for the student in this exercise."
 
+#: exercise/templates/exercise/staff/inspect_submission.html
+msgid "GRADING_MODE_BEST_EXPLAIN"
+msgstr "Choosing highest grade"
+
+#: exercise/templates/exercise/staff/inspect_submission.html
+msgid "GRADING_MODE_LAST_EXPLAIN"
+msgstr "Choosing latest submission"
+
```

Comment:
Is it clear what "choosing" means? To me, "choosing" makes it sound like the user is expected to choose something.

Reply:
On a second thought I decided to use the existing translations "Grading mode" and "Best/Last Submission", which should be clear enough for the course staff. Anyway these texts were my own idea, and not a teacher request, so we could just leave it out as well.

Reply:
Nice! I like this a lot.